blocklylib-core/src/main/java/com/google/blockly/android/AbstractBlocklyActivity.java

```diff
@@ -807,14 +617,6 @@ protected boolean onBackToCloseNavMenu() {
         return false;
     }
 
-    /**
-     * @return True if the action was handled to close a previously open (and closable) flyout.
-     *         Otherwise false.
-     */
-    protected boolean onBackToCloseFlyouts() {
-        return mController.closeFlyouts();
-    }
-
     private class DefaultVariableCallback extends BlocklyController.VariableCallback {
```

Comment:
As a TODO we should pull this out into it's own file. Maybe DialogVariableCallback?